lib/src/modules/conditional.dart

```diff
@@ -890,6 +889,43 @@ enum ConditionalType {
   priority
 }
 
+/// Shorthand for a [Conditional] inside a [Case] block.
```

Comment:
Backwards, this is a `Case` inside of a `Conditional`.    Also, "Returns a [Logic] signal based on the expression and conditions." doesn't really elaborate on how to use it or what it will do.  Imagine a user is reading this doc comment in order to learn what this function does and what to expect it will do.  You can even include some example code in the doc comment.

Also, it's usually best to keep 1 line empty after the first line.

Here's some tips on writing doc comments:
https://dart.dev/effective-dart/documentation